pages/models/Capacitors/CapacitorBank/index.md

```diff
@@ -0,0 +1,124 @@
+$$\|V_R\| = \frac{1}{2} \|V_S\| + \frac{1}{2} \|V_S\| \sqrt(1 - \frac{4 X Q_R}{\|V_S\|^2}) $$
+</div>
+       
+To maintain the level of voltage, the shunt element has to maintain the same $$Q_R$$ by absorbing or injecting the reactive power to compensate the load fluctuations: $$Q_C = Q_R - Q_D$$. If the load is higher than $$Q_R$$, then the shunt element will need to absorb reactive power, which means it will have to be inductive. If the load is lower than $$Q_R$$, then the shunt element will need to inject reactive power, which means it will have to be capacitive. The model studies the second case.
+
+### Capacitor bank dimensioning
+
+The capacitor bank is considered to be connected to the end of a transmission line at its voltage level $$\|V_R\|$$. The following schematic represents the simplified circuit:
+
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/Capacitors/CapacitorBank/CapBank_scheme.svg' | relative_url }}"
+     alt="Capacitor bank scheme"
+     style="float: center; margin-right: 10px;" />
+</div>
+<div align = 'center'>
+Figure 1: Capacitor bank scheme
+</div>
+<br>
+
+In this circuit, the current passing through the capacitor bank is given by:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$I_C = j \frac{\|V_R\|}{\sqrt(3) X_C}$$
+
+</div>
+
+where $$X_C$$ is the capacitive reactance of the capacitor bank. To calculate the reactive power injected the following equation is used:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$jQ_C = 3 \frac{\|V_R\|}{\sqrt(3)} (-I_C^*)$$
+$$Q_C = \frac{\|V_R\|^2}{X_C}$$
+
+</div>
+
+To dimension the capacitive bank, the capacitive reactance is calculated from the desired reactive power injection and the voltage level that has to be maintained, calculated using the relation derived in the previous section.
+
+
+## Open source implementations
+
+This model has been successfully implemented in :
+
+
+| Software      | URL | Language | Open-Source License | Last consulted date | Comments |
+| --------------| --- | --------- | ------------------- |------------------- | -------- |
+| dynawo | [Link](https://github.com/dynawo/dynawo/blob/master/dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunts/ShuntB.mo) | modelica | [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)  | 24/05/2024 | - |
+| Andes | [Link](https://github.com/CURENT/andes/blob/master/andes/models/shunt/shunt.py) | python | [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html)  | 24/05/2024 | - |
+| Andes | [Link](https://github.com/e2nIEE/pandapower/blob/develop/pandapower/converter/cim/cim2pp/converter_classes/shunts/linearShuntCompensatorCim16.py) | python | [3-clause BSD](https://pandapower.readthedocs.io/en/v2.0.1/about/license.html)  | 24/05/2024 | - |
```

Comment:
The last one in the software column should be updated to pandapower instead of Andes.